cluster-scope/overlays/moc/zero/kustomization.yaml

```diff
@@ -1,184 +1,184 @@
----
```

Comment:
Ah okay so the docs say to use `kustomize edit add` but it seems like it will reformat the whole file. Can you revert the change on this file, and add the resources manually (alphabetically sorted)? So the diff isn't this large? 

You would just need to add the `../../../base/core/namespaces/adoptium-project` bit under the `resources` field.

Reply:
@tumido @4n4nd wdyt ? should we discourage the use of `kustomize edit add` in the docs so it doesn't mess with file formatting?

Reply:
I have suggested @Gregory-Pereira use the new opfcli. That way he could help us with the docs as well.

Reply:
he'll still need to add the namespace manually here.